setup.py

```diff
@@ -14,11 +14,12 @@
     changes = changes_file.read()
 
 install_requires = [
-    "threedigrid",
-    # "gdal",  # assumed available via virtualenv --system-site-packages
+    "h5py",
+    "netCDF4",
```

Comment:
Heb je echt netCDF4 lib nodig? Deze werkt niet altijd heel lekker als je hem naast h5py gebruikt....

Reply:
Richard heeft netCDF erin gebakken, maar ik vond het wel lelijk om het op basis van trial en error te doen. Ik weet niet of die problemen er ook zijn op windows. Lijkt me mooie testcase. En anders kan ik hem wel omschrijven naar h5py.

Reply:
Oke, lijkt me dan voor nu prima.